build_all.sh

```diff
@@ -50,8 +50,8 @@ do
 	echo "                    Pushing Docker Images for Version ${ver}                   "
 	echo "                                                                               "
 	echo "==============================================================================="
-	cat push_commands.sh
-	./push_commands.sh
+#	cat push_commands.sh
```

Comment:
Why is this commented out?

Reply:
For test on my fork, I enabled travis build there, but actually it seems that files are not regenerated (no hadolint output) - need to further investigate that

Reply:
And generally need to change idea, as probably travis build is leftover, and you use those scripts somewhere on jenkins build farm, so need to `wget` hadolint binary and use it in that way instead of via docker